README.md

```diff
@@ -85,43 +85,23 @@ Make sure nginx is deployed in you K8s. Otherwise you can deploy it using
 make nginx-up
 ```
 
-or by running
-
-``` sh
-kubectl apply -f kube/nginx.yaml
-```
-
-If you launching atlas-contacts-app for the first time you need to create `contacts` namespace for it in Kubernetes. This can be done by running
-
-``` sh
-kubectl apply -f kube/ns.yaml
+##### Deployment
+Upload atlas-contacts-app images into minikube
 ```
-
-or
-
-``` sh
-kubectl create ns contacts
+make push-minikube
 ```
 
-##### Deployment
-
 To deploy atlas-contacts-app use
 
 ``` sh
 make up
 ```
 
-or as alternative you can run
-
-``` sh
-kubectl apply -f kube/kube.yaml
-```
-
 To deploy authN stub, clone atlas-stubs repo and then execute deployment script inside authn-stub package:
```

Comment:
can we just pull a docker image of the auth stub instead of having to build it locally?